### DISCUSSION TITLE
Model File Exist Error When Running on Colab

### DISCUSSION THREAD

**Discussion by mikenoname-github** [openai/whisper#1618] [Q&A] [answered]
I have uploaded the V2-Large model to my Google Drive and am trying to call it from a Colab instance using:

!whisper "/content/drive/MyDrive/Transcription/PD3.m4a" --model large --language English --model_dir "/content/drive/MyDrive/Whisper Model/large-v2.pt" --output_format txt --condition_on_previous_text False

It results in the following output:

Traceback (most recent call last):
  File "/usr/local/bin/whisper", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.10/dist-packages/whisper/transcribe.py", line 444, in cli
    model = load_model(model_name, device=device, download_root=model_dir)
  File "/usr/local/lib/python3.10/dist-packages/whisper/__init__.py", line 131, in load_model
    checkpoint_file = _download(_MODELS[name], download_root, in_memory)
  File "/usr/local/lib/python3.10/dist-packages/whisper/__init__.py", line 49, in _download
    os.makedirs(root, exist_ok=True)
  File "/usr/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/content/drive/MyDrive/WhisperModel/large-v2.pt'

I must be doing something wrong, but can't see what it is. Can anyone help?

Thank you!

**Comment by phineas-pta** [comment]
remove `--model-dir "....."`

**Reply by mikenoname-github** [reply]
But then won't that defeat the purpose of my calling the model from my Drive? Or am I missing something?

**Reply by phineas-pta** [reply]
why u put the model in your drive ?

**Reply by glangford** [reply, marked as answer]
Agreed with @phineas-pta , it isn't really necessary to put the model on your Google Drive. You can just install it remotely each time.

That said - if for some reason you really do want to keep models on your Google Drive, there is an error in the command line. `--model_dir` requires a *directory*. So the way to do this would be

`--model_dir "/content/drive/MyDrive/Whisper Model/"`

(without large-v2.pt at the end)

**Reply by mikenoname-github** [reply]
Thanks Glenn, that was the piece that I missed!

And thanks Phan for your help. I always do install it remotely each time I use it, but I wanted to know if I could control the model being used without having to fetch it from a resource that I don't have control over.

**Reply by phineas-pta** [reply]
the model file is 3gb = u lost 20% of free 15gb

not sure what u want to control, the only download link for `.pth` file is by openai

**Comment by cod3xpert** [comment]
@mikenoname-github you have passed --model_dir "/content/drive/MyDrive/Whisper Model/large-v2.pt" 
here you just need to pass model_dir which is "/content/drive/MyDrive/Whisper Model"
remove large-v2.pt it will auto pick the model file based on --model large ( model name )
